src/test/java/racinggame/domain/WinnerTest.java

```diff
@@ -0,0 +1,28 @@
+package racinggame.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class WinnerTest {
+
+    @Test
+    @DisplayName("우승자가 여러 명일 경우 모두 출력한다")
+    void multipleWinnersAreAllAnnounced() {
+        Car pobi = new Car("pobi");
+        Car crong = new Car("crong");
+        Car honux = new Car("honux");
+
+        pobi.move(5);
+        crong.move(5);
+        honux.move(3);
```

Comment:
```suggestion
        Car pobi = new Car("pobi", 5);
        Car crong = new Car("crong", 5);
        Car honux = new Car("honux", 3);
```
위와 같이 position 값을 지정할 수 있는 생성자를 추가해 보면 어떨까?